.github/workflows/vertexai.yml

```diff
@@ -15,6 +15,41 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-mock-responses-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Clone mock responses
+      run: scripts/update_vertexai_responses.sh
+    - name: Find cloned and latest versions
+      run: |
+        echo "current_tag=$(git describe --tags | awk -F'/' '{print $NF}')" >> $GITHUB_ENV
+        echo "latest_tag=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/FirebaseExtended/vertexai-sdk-test-data.git | tail -n1 | awk -F'/' '{print $NF}')" >> $GITHUB_ENV
```

Comment:
Why you need the last `awk` command?

Reply:
The output before `awk` includes the commit hash and looks like this:
```
0b55b397d27706407eb4062dd70f18cad47edcfc        refs/tags/v1.0
```

I'm using the `awk` command to extract the tag `v1.0` so that it can be compared with the output of `git describe --tags`.

But I see that the `awk` command above with `git describe --tags` is unnecessary.

Reply:
Yes, thanks. I meant to comment on the command above.